tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -116,6 +127,14 @@ object NativeConfig {
     def withCompileOptions(value: Seq[String]): NativeConfig =
       copy(compileOptions = value)
 
+    def withTargetTriple(value: Option[String]): NativeConfig =
+      copy(targetTriple = value)
+
+    def withTargetTriple(value: String): NativeConfig = {
+      val optValue = if (value.trim().isEmpty()) None else Some(value)
+      withTargetTriple(optValue)
+    }
```

Comment:
This is too fancy. The overload with a `String` is just a helper to avoid the `Some()`. It should just be
```suggestion
    def withTargetTriple(value: String): NativeConfig =
      withTargetTriple(Some(value))
```
In a "configuration language", this would make sense. But here we are defining a configuration API.

Reply:
Thanks, this is an oversight (copy) from a previous version where I had the `sbt` setting and when not set, `""` was the value passed from the setting.